src/views/PeopleSearch/components/PeopleSearchResult/index.js

```diff
@@ -143,6 +143,49 @@ export default class PeopleSearchResult extends Component {
           <Divider />
         </>
       );
+    } else if (size === 'largeImages') {
+      /*** Enlarged Images ***/
+      return (
+        <>
+          <Divider />
+          <Link className="gc360-link" to={`profile/${Person.AD_Username}`}>
+            <Grid
+              container
+              alignItems="center"
+              justify="center"
+              spacing={2}
+              style={{
+                padding: '1rem',
+              }}
+            >
+              <Grid item>
+                <IMG
+                  className="people-search-avatar-large"
+                  src={`data:image/jpg;base64,${this.state.avatar}`}
+                  alt=""
+                  noLazyLoad="true"
+                  placeholderColor="#eeeeee"
+                />
+              </Grid>
+              <Grid
+                item
+                style={{
+                  // a set width is necessary to keep profile images in line
+                  // while maintaining center alignment
+                  width: '260px',
+                }}
+              >
```

Comment:
There's an easier and better way to keep the images centered without setting a static width:
```suggestion
              <Grid item xs={6} container justify="flex-end">
                <IMG
                  className="people-search-avatar-large"
                  src={`data:image/jpg;base64,${this.state.avatar}`}
                  alt=""
                  noLazyLoad="true"
                  placeholderColor="#eeeeee"
                />
              </Grid>
              <Grid item xs={6}>
```

Reply:
I didn't catch that part when I was copying from the mobile view, thanks for finding that!